### PR TITLE
[CS-Imaginate] support multiple targets, add inker pic

### PR DESCRIPTION
**Pull request by inkerchaoshi**
1. support multiple targets which we fix in circuit
2. add inker pic
3. random background color

related PR: [https://github.com/customink/circuit/pull/608](url)

### CODE REVIEW

File: js/wow.js [hunk 1]
```diff
@@ -33,12 +39,25 @@ function clearSlides() {
 }
 
 function addSlide(wow) {
-  var title = $('<h2>').html("Wow " + wow.target);
+  var names = wow.targets.map(function(target){
+    return target.name;
+  }).join(",");
```
Comment (cmar) [inline js/wow.js]: What do you think about adding a space after the comma?

Comment (inkerchaoshi) [reply js/wow.js]: good idea, will do!

**Comment by rypit** [comment]
Sorry, not too familiar with this app. Code looks good though!

**Review by cmar — APPROVED** [review]
Thanks for upgrading the app. Looks great.

**Pull request merged**